mis_builder/report/mis_report_instance_xlsx.py

```diff
@@ -142,6 +144,20 @@ def generate_xlsx_report(self, workbook, data, objects):
                                          len(cell.val_rendered or ''))
             row_pos += 1
 
+        # Add date/time footer
+        row_pos += 1
+        footer_format = workbook.add_format({
+            'italic': True, 'font_color': '#202020', 'size': 9})
+        now = datetime.now()
+        lang_model = self.env['res.lang']
+        lang = lang_model._lang_get(self.env.user.lang)
+        date_format = lang.date_format
+        time_format = lang.time_format
+        create_date = _("Generated on {} at {}").format(
+            now.strftime(date_format),
+            now.strftime(time_format))
```

Comment:
I think there something better to get now in the user timezone and language. Maybe `tools.fromat_tz()`?

Reply:
I'm not sure that we want to depend on a function declared in addons.mail.models.mail_template?
We can still use:
```suggestion
utc_now = datetime.now()
tz = pytz.timezone(self.env.user.tz or "UTC")
now = pytz.utc.localize(utc_now).astimezone(tz)
create_date = _("Generated on {} at {}").format(
    now.strftime(date_format),
    now.strftime(time_format))
```

Reply:
@apineux indeed depending on the mail module is not a good idea.
I think `fields.Datetime.context_timestamp` is the way to go.